app/views/merge_requests/user_outcomes.html.erb

```diff
@@ -10,7 +10,7 @@
 </h1>
 
 <% unless @merge_request.status == "request_merged" || @merge_request.status == "processing" %>
-  <h2 class="govuk-heading-m">19 users after merge</h2>
+  <h2 class="govuk-heading-m"><%= @merge_request.total_visible_users_after_merge %> users after merge</h2>
```

Comment:
Should we take into account the case of having a singular user and displaying "users" or "user"?

Reply:
For sure, good spot! Should be updated now!